dashboard/config/foorm/forms/surveys/pd/ayw_workshop_pre_survey.0.json

```diff
@@ -0,0 +1,1371 @@
+        },
+        {
+         "value": "14_Automating_Data_Decisions",
+         "text": "14. Automating Data Decisions"
+        },
+        {
+         "value": "15_Project_-_Make_a_Recommendation",
+         "text": "15. Project - Make a Recommendation"
+        },
+        {
+         "value": "16_Post-Project_Test",
+         "text": "16. Post-Project Test"
+        },
+        {
+         "value": "17_CS_Discoveries_Post_Course_survey",
+         "text": "17. CS Discoveries Post Course survey"
+        }
+       ]
+      },
+      {
+       "type": "dropdown",
+       "name": "csd_unit_6_lessons",
+       "visibleIf": "{csd_unit} = 'Unit_6'",
+       "title": "Which lesson from unit {csd_unit} did you most recently complete. (If not sure, make a best guess)",
+       "choices": [
+        {
+         "value": "1_Innovations_in_Computing",
+         "text": "1. Innovations in Computing"
+        },
+        {
+         "value": "2_Designing_Screens_with_Code",
+         "text": "2. Designing Screens with Code"
+        },
+        {
+         "value": "3_The_Circuit_Playground",
+         "text": "3. The Circuit Playground"
+        },
+        {
+         "value": "4_Input_Unplugged",
+         "text": "4. Input Unplugged"
+        },
+        {
+         "value": "5_Board_Events",
+         "text": "5. Board Events"
+        },
+        {
+         "value": "6_Getting_Properties",
+         "text": "6. Getting Properties"
+        },
+        {
+         "value": "7_Analog_Input",
+         "text": "7. Analog Input"
+        },
+        {
+         "value": "8_The_Program_Design_Process",
+         "text": "8. The Program Design Process"
+        },
+        {
+         "value": "9_Project:_Make_a_Game",
+         "text": "9. Project: Make a Game"
+        },
+        {
+         "value": "10_Arrays_and_Color_LEDs",
+         "text": "10. Arrays and Color LEDs"
+        },
+        {
+         "value": "11_Making_Music",
+         "text": "11. Making Music"
+        },
+        {
+         "value": "12_Arrays_and_For_Loops",
+         "text": "12. Arrays and For Loops"
+        },
+        {
+         "value": "13_Accelerometer",
+         "text": "13. Accelerometer"
+        },
+        {
+         "value": "14_Functions_with_Parameters",
+         "text": "14. Functions with Parameters"
+        },
+        {
+         "value": "15_Circuits_and_Physical_Prototypes",
+         "text": "15. Circuits and Physical Prototypes"
+        },
+        {
+         "value": "16_Project:_Prototype_an_Innovation",
+         "text": "16. Project: Prototype an Innovation"
+        },
+        {
+         "value": "17_Post-Project_Test",
+         "text": "17. Post-Project Test"
+        },
+        {
+         "value": "18_CS_Discoveries_Post_Course_survey",
+         "text": "18. CS Discoveries Post Course survey"
+        }
+       ]
+      }
+     ]
+    },
+    {
+     "type": "comment",
+     "name": "question3",
```

Comment:
from here below, need to update question names

Reply:
Fixed